notebooks/running_jobs.ipynb

```diff
@@ -17,41 +31,65 @@
+    "* __sequence_id__ (Uniprot ID or AC): Identifier of your sequence. Will be automatically fetched from Uniprot.\n",
+    "* __region__: Region of the protein to run, in uniprot numbering. Leave blank for full sequence or put [start_pos, stop_pos]\n",
+    "\n",
+    "### Additional parameters to modify (non-exhaustive):\n",
+    "\n",
+    "The following additional parameters are the most important ones to consider modifying when running your job. See example configuration files for full parameter explanations:\n",
+    "\n",
+    "1) In \"alignment\" section:\n",
+    "* __domain_threshold__ (bitscore or evalue): Inclusion threshold for sequence alignment. 0.5 is a good starting point for bitscore, e.g. use 0.3 for more inclusive and 0.7 or 0.8 for less inclusive alignment\n",
+    "* __sequence_threshold__ (bitscore or evalue): typically should be same as domain_threshold\n",
+    "* __use_bitscores__: default true, set to false to use evalues for domain_threshold and sequence_threshold\n",
+    "* __iterations__: number of jackhmmer iterations\n",
+    "* __database__: uniref, uniprot (see databases section for options or to modify path)\n",
+    "* __seqid_filter__: Filter alignment at x% sequence identity. Leave empty or set to 100 not to filter any sequences.\n",
+    "* __minimum_sequence_coverage__: Only keep sequences that align to at least x% of the target sequence (formerly called \"-f\" in buildali)\n",
+    "* __minimum_column_coverage__: Only include alignment columns with at least x% residues (rather than gaps) during model inference. (formerly called \"-m\" in buildali, but now the other way round: -m 30 corresponds to minimum_column_coverage=70)\n",
+    "\n",
+    "2) in \"couplings\" section:\n",
+    "* __theta__: Clustering threshold for downweighting redudant sequences (Meff computation). For example, 0.8 will cluster sequencesat a 80% sequence identity cutoff (note that this is 1 - theta used in previous pipelines)\n",
+    "* __iterations__: how many plmc iterations to run\n",
+    "* __ignore_gaps__: exclude gaps from EC calculation\n",
+    "* __save_model__: save binary file with model parameters or not. These files can be very large.\n",
+    "\n",
+    "3) in \"compare\" section:\n",
+    "* __by_alignment__: If True, structures for comparison will be identified by homology search; otherwise use only structures for given sequence_id (must be UniProt ID or AC)\n",
+    "* __pdb_alignment_method__: jackhmmer or hmmsearch. Method for searching the PDB sequence database for homologous structures. Jackhmmer is more stringent, hmmsearch is more lenient. \n",
+    "* __pdb_ids__: Used if by_alignment is False. If pdb_ids is None, compares to all PDB structures for given sequence_id. If list of PDB IDs, compare to that subset of structures only.\n",
```

Comment:
Actually this will be used no matter if alignment or lookup by Uniprot ID/AC is used - it just acts as a filter on the structure list independent of how it was made